CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ include(CheckLibraryExists)
 find_package(LibUV REQUIRED)
 include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 
-find_package(Msgpack REQUIRED)
+find_package(Msgpack 1.0.0 REQUIRED)
```

Comment:
Good question.  No, it isn't but I think I see how to make it so that is. :) Currently, FindMsgpack.cmake will also need to update the `pkg_search_module` call, but it looks like `find_package` exports some variables we can use there.